modules/app/src/main/resources/assets/js/main.ts

```diff
@@ -410,22 +412,37 @@ function getTheme(): string {
 async function startContentBrowser() {
     await import ('lib-contentstudio/app/ContentAppPanel');
     const AppWrapper = (await import ('lib-contentstudio/app/AppWrapper')).AppWrapper;
-    const baseApps = [new ContentApp(), new SettingsApp()];
+    const baseApps = [new ContentApp()];
     const url: string = window.location.href;
     const commonWrapper = new AppWrapper(baseApps, getTheme());
-    const baseAppToBeOpened = baseApps.find((baseApp) => url.endsWith(`/${baseApp.getUrlPath()}`));
+    const baseAppToBeOpened = url.indexOf('#') < 0 ? baseApps[0] :
+                              baseApps.find((baseApp) => url.endsWith(`/${baseApp.getUrlPath()}`));
 
     if (baseAppToBeOpened) {
         commonWrapper.selectApp(baseAppToBeOpened);
     } else {
-        commonWrapper.onAppAdded((app) => {
-            if (!commonWrapper.hasAppSelected() && url.endsWith(`/${app.getUrlPath()}`)) {
-                commonWrapper.selectApp(app);
+        commonWrapper.onItemAdded((item) => {
```

Comment:
Use `(item: App | Widget)` and check for instance type instead of checking for properties that might disappear or be renamed (`item['getUrlPath']`).